tests/test_x509_ext.py

```diff
@@ -112,3 +129,94 @@ def test_unsupported_extension(self, backend):
         )
         extensions = cert.extensions
         assert len(extensions) == 0
+
+
+@pytest.mark.requires_backend_interface(interface=RSABackend)
+@pytest.mark.requires_backend_interface(interface=X509Backend)
+class TestRSABasicConstraintsExtension(object):
```

Comment:
Yeah, I'm only asking about changing the name.

On Sat, Mar 28, 2015 at 1:41 PM, Paul Kehrer <notifications@github.com>
wrote:

> In tests/test_x509_ext.py
> <https://github.com/pyca/cryptography/pull/1769#discussion_r27346799>:
>
> > @@ -112,3 +129,94 @@ def test_unsupported_extension(self, backend):
> >          )
> >          extensions = cert.extensions
> >          assert len(extensions) == 0
> > +
> > +
> > +@pytest.mark.requires_backend_interface(interface=RSABackend)
> > +@pytest.mark.requires_backend_interface(interface=X509Backend)
> > +class TestRSABasicConstraintsExtension(object):
>
> That's the only relation. We need to keep the RSABackend mark but we
> could remove the RSA component of the name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1769/files#r27346799>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084